test/0.0.1-rc.16/domain.spec.ts

```diff
@@ -551,6 +552,7 @@ describe('Domain Layer', () => {
         address: addresses.ContributionReward.toLowerCase(),
         name: 'ContributionReward',
         numberOfBoostedProposals: '0',
+        numberOfExpiredInQueueProposals: '0',
```

Comment:
is there a test which actually test that numberOfExpiredInQueueProposals increments sand decrements ?

Reply:
I guess no. So I can add a proposal in the domain test.

Reply:
please add